src/main/java/de/tum/in/www1/artemis/config/Constants.java

```diff
@@ -82,6 +82,8 @@
 
     public static final String TEST_CASES_CHANGED_NOTIFICATION = "The test cases of this programming exercise were updated. The student submissions should be build and tested so that results with the updated settings can be created.";
 
+    public static final String TEST_CASES_DUPLICATE_NOTIFICATION = "There are duplicated test cases in this programming exercise. All test cases should be unique and not have the same name. The following test cases are duplicated: ";
```

Comment:
```suggestion
    public static final String TEST_CASES_DUPLICATE_NOTIFICATION = "There are duplicated test cases in this programming exercise. All test cases have to be unique and cannot have the same name. The following test cases are duplicated: ";
```

Reply:
done in 8f8252aea4715d041d5c3aad1c820371083b6655